@here/harp-transfer-manager/src/TransferManager.ts

```diff
@@ -86,8 +86,10 @@ export class TransferManager implements ITransferManager {
     ): Promise<Response> {
         try {
             const response = await fetchFunction(url, init);
-            if (response.status !== 503 || retryCount > maxRetries) {
+            if (response.ok) {
```

Comment:
this is changing the functionality, as it will neither return not throw for respomses with not "ok" status

Reply:
@FraukeF , I don't know why I pushed this, I have pushed the fix.